blockchain/validators/src/metadata/avr/config.rs

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! Facilities for creating an AVR verifier object from a configuration file.
+
+use crate::{
+    error::ParseError,
+    metadata::avr::{AvrValidationRecord, AvrValidator},
+    ValidationError,
+};
+use mc_attest_core::{serial, VerificationReport};
+use mc_attest_verifier::{Verifier, DEBUG_ENCLAVE};
+use mc_blockchain_types::BlockIndex;
+use mc_common::ResponderId;
+use mc_crypto_keys::Ed25519Public;
+use serde::{Deserialize, Serialize};
+use serde_json;
+use std::{collections::HashMap, ffi::OsStr, fs, path::Path};
+use toml;
+use TryFrom;
+
+const IAS_ANCHORS: &[&str] = &[include_str!(
```

Comment:
Yeah it might be. And I'm not entirely sure yet what the right solution is? 

It seems to me that if we're trying for Avr Verification - we do want to try to ensure that the AVR was generated from intel's chain of trust? Otherwise people could be feeding data that mimics AVRs from Intel signed enclaves with a false "block signing key" that matches that in the fake AVR.

For these tests we can switch to test data so production certificates don't have to be on chain, and potentially disable this step in test deploys?

Reply:
This is what's been done! Very helpful suggestion, thank you.